src/utils/uuid.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+const BaseUUID = `${1e7}-${1e3}-${4e3}-${8e3}-${1e11}`;
+
+function uuidPart(numberString: string): string {
+  const a = Number(numberString);
+  return (a ^ ((Math.random() * 16) >> (a / 4))).toString(16);
+}
+
+function uuid(): string {
```

Comment:
Er dette for å unngå en ny dep?

Reply:
Litt det, men også at `@ndla/util` (som allerede har en `uuid` funksjon) har peer-dependency på react. Og jeg har veldig lite lyst å dra inn react i graphql-api :smile: